CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Main (unreleased)
 
+## v239 (2022/03/02)
```

Comment:
```suggestion
## v239 (2022/03/03)
```

Reply:
I did the rollback/deploy yesterday, it's a retroactive changelog